docs/docker-testnet.md

```diff
@@ -66,12 +66,12 @@ docker run -d -p 3013:3013 -e PEERS_ADDRESS_0=http://31.13.248.103:3013/ aetrnty
 
 ## Persisting Data
 
-To persist blockchain data and node keys between container runs, use [Docker volumes](https://docs.docker.com/engine/admin/volumes/volumes/). Replace `~/.aeternity/.chain` with location of your choice.
+To persist blockchain data and node keys between container runs, use [Docker volumes](https://docs.docker.com/engine/admin/volumes/volumes/). Replace `~/.aeternity/data` with location of your choice.
```

Comment:
Is the default db_path = `data` now ?
If so, IMO we should also change the example configuration files

Reply:
Do you mean https://github.com/aeternity/epoch/blob/999c46392991310c8dfcfda83f8fda152f5e537e/apps/aeutils/test/data/epoch_full.yaml#L43 ?

Reply:
Not sure we have to change anything here. The mnesia database will place itself in whatever directory we want, and will coexist there with the old persistence files.

Reply:
> Not sure we have to change anything here.

@uwiger  For sure a constraint we have in `-v ~/.aeternity/data:/home/epoch/node/data` is that *with the config that the Docker image uses* the DB dir must be `/home/epoch/node/data`.

Reply:
Linked this from https://www.pivotaltracker.com/story/show/154636528/comments/185105839